config/locales/errors/en.yml

```diff
@@ -115,7 +115,8 @@ en:
       attestation_error: Sorry, but your security key doesn’t appear to be a FIDO
         security key.  Please make sure your device is listed at
         https://fidoalliance.org/certification/fido-certified-products/ and if
-        you believe the error is ours, please contact us at hello@login.gov.
+        you believe the error is ours, please contact at
+        https://login.gov/contact/
```

Comment:
Do you think it would make sense to make these URLs interpolated? ex `%{fido_url}` and `%{contact_url}` and then we can use like `MarketingSite` class that auto inserts the locale?

Reply:
yup we already ahve the contact url so i updated and pushed up that change .